clinvar/src/main/scala/org/broadinstitute/monster/etl/clinvar/models/output/SCVTrait.scala

```diff
@@ -46,31 +45,22 @@ object SCVTrait {
     val nameWrapper = rawTrait.extract("Name", "ElementValue")
     val nameType = nameWrapper.flatMap(_.extract("@Type")).map(_.str)
 
-    val allXrefs = MsgTransformations.popAsArray(rawTrait, "XRef")
+    val allXrefs = MsgTransformations.popAsArray(rawTrait, "XRef").map { xref =>
```

Comment:
Nitpick: could just be `MsgTransformations.popAsArray(rawTrait, "XRef").map(XRef.fromRawXRef)`